src/pull-request/pull-request.service.ts

```diff
@@ -146,19 +156,49 @@ export class PullRequestService {
 
   public convertPatch(
     patchFile: string,
-    singleKbRepoName: string,
+    keyboardId: string,
+    prefix: string,
   ): Observable<string> {
-    const prefix = this.getNewPathPrefix(singleKbRepoName);
+    prefix = this.getNewPathPrefix(keyboardId, prefix);
     return readFile(patchFile).pipe(
       map(data => this.convertPatchFile(prefix, data.toString())),
       switchMap(content => writeFile(patchFile, content)),
       map(() => patchFile),
     );
   }
 
-  private getNewPathPrefix(singleRepoName: string): string {
+  private getNewPathPrefix(keyboardId: string, prefix: string): string {
     // git uses / as directory separator, even on Windows
-    return `release/${singleRepoName.substring(0, 1)}/${singleRepoName}`;
+    return `release/${this.getPrefix(keyboardId, prefix)}/${keyboardId}`;
+  }
+
+  public getPrefix(keyboardId: string, prefix: string): string {
+    if (!keyboardId) {
+      return keyboardId;
+    }
+
+    if (prefix) {
+      return prefix;
+    }
+
+    const specialPrefixes = [
```

Comment:
This will need to be built from the keyboards repo folder names as it does change frequently.

Reply:
https://github.com/keymanapp/developer.keyman.com/issues/73